.github/actions/prepare-ci/action.yml

```diff
@@ -0,0 +1,36 @@
+name: 'Prepare CI'
+description: 'Prepares an environment to run a set of Eradiate tests'
+outputs:
+  mitsuba-cache-hit:
+    description: "Mitsuba cache hit"
+    value: ${{ steps.cache.outputs.cache-hit }}
+runs:
+  using: "composite"
+
+  steps:
+  - name: Install Python
+    uses: actions/setup-python@v5
+    with:
+      python-version: "3.12"
+  - name: Install Chrome
+    uses: browser-actions/setup-chrome@v1
+  - name: Install dependencies
+    shell: bash
+    run: |
+      python -m pip install --upgrade pip
+      pip install .[doc,recommended,test,dev]
```

Comment:
Something we might want to discuss (not here, let's focus on the pipeline for this PR): this does not use a reproducible testing env. Whether the testing env should be frozen has pros and cons, and I don't have a settled opinion on the matter.

Reply:
Sure